src/Layer/Layer.js

```diff
@@ -53,6 +53,7 @@ class Layer extends THREE.EventDispatcher {
     constructor(id, type, config = {}) {
         super();
 
+        // ??
```

Comment:
What is that ?

Reply:
I create a issue to explain my comment

Reply:
see #921

Reply:
Thanks, but please remove these comments, or replace them by linking to the issue